data/lm/README.md

```diff
@@ -0,0 +1,46 @@
+lm.binary was generated from the LibriSpeech normalized LM training text, available [here](http://www.openslr.org/11), following this recipe (Jupyter notebook code):
```

Comment:
Should we format this a bit better?

Reply:
How?

Reply:
Was just thinking at least title, headings, and the like.